test/lib/util/token_bucket_test.exs

```diff
@@ -0,0 +1,17 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.TokenBucketTest do
+  use Croma.TestCase
+  alias Croma.Result, as: R
+  alias Antikythera.TokenBucket
```

Comment:
Since you are using `Croma.TestCase`, you can delete this line.

Reply:
Thank you.
Fixed in 593f228af9bed136734af602491a76dc06496301